<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reply_widget</class>
 <widget class="QWidget" name="reply_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>19</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#btn {
	background: none;
	border: none;
}

#label_username {
	margin-bottom: 0px;
	font-size: 9pt;
	font-weight: 600;
	color: black;
}

#label_message {
	color: #B0B0B0;
	font-size: 9pt;
}

#label_message[highlight=&quot;true&quot;] {color: white;}

#line {background-color: #808080;}
#line[highlight=&quot;true&quot;] {background-color: white;}</string>
  </property>
  <layout class="QVBoxLayout" name="layout_widget">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>48</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="FrameButton" name="btn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::TabFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="layout_reply" stretch="0,0,0,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="line">
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MaskedImage" name="icon_avatar">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../resources/index.qrc">:/icons/avatar</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string notr="true">Username</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_message">
        <property name="text">
         <string notr="true">Message text</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameButton</class>
   <extends>QFrame</extends>
   <header>widgets.common</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaskedImage</class>
   <extends>QLabel</extends>
   <header>widgets.common</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/index.qrc"/>
  <include location="../../../resources/index.qrc"/>
 </resources>
 <connections/>
</ui>
